src/components/workspace-utils.js

```diff
@@ -38,6 +38,25 @@ export const useWorkspaces = () => {
   return { workspaces, refresh, loading }
 }
 
+export const useWorkspaceDetails = ({ namespace, name }, fields) => {
+  const [workspace, setWorkspace] = useState()
+
+  const [loading, setLoading] = useState(true)
+  const signal = useCancellation()
+
+  const refresh = _.flow(
+    withErrorReporting('Error loading workspace'),
```

Comment:
It would be good to make the error message more descriptive (perhaps 'Error loading workspace details').

Reply:
Revised in [84f28aa](https://github.com/DataBiosphere/terra-ui/pull/2886/commits/84f28aad7dcbc78fae67b1496b0f6522496e5d9e)